hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java

```diff
@@ -358,6 +338,30 @@ private boolean getFilesForArchive(DBCheckpoint checkpoint,
         compactionLogDir.getOriginalDir().toPath());
   }
 
+  private void logTotals(
```

Comment:
nit: rename is to `logEstimatedTarballSize`

Reply:
```suggestion
  private void logEstimatedTarballSize(
```

Reply:
Updated.